### PR TITLE
Add file upload to desktop close workOrder form

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Summary of Changes

Add file upload functionality to close work order form on desktop.

image

image

image

image

image

image

### CODE REVIEW

File: src/components/WorkOrder/Photos/hooks/useFileUpload.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const useFileUpload = (workOrderReference, onSuccess) => {
     const uploadResult = await uploadFiles(
       files,
       workOrderReference,
+      '',
```
Comment (Duslerke) [inline src/components/WorkOrder/Photos/hooks/useFileUpload.js:30]: Just confirming: the `uploadGroupLabel` will get set elsewhere after the upload?

File: src/components/WorkOrders/CloseWorkOrderByProxy.js [hunk 2]
```diff
@@ -281,8 +301,14 @@ const CloseWorkOrderByProxy = ({ reference }) => {
...
     <>
-      {loading ? (
-        <Spinner />
+      {loadingStatus ? (
+        <div
+          className="govuk-body"
+          style={{ display: 'flex', alignItems: 'center' }}
+        >
+          <Spinner />
+          <span style={{ margin: '0 0 0 15px' }}>{loadingStatus}</span>
```
Comment (Duslerke) [inline src/components/WorkOrders/CloseWorkOrderByProxy.js:310]: Clever!

File: src/components/WorkOrder/MobileWorkingWorkOrderView.js [hunk 3]
```diff
@@ -172,6 +172,7 @@ const MobileWorkingWorkOrderView = ({ workOrderReference }) => {
           files,
           workOrderReference,
           'Closing work order',
+          '',
```
Comment (humulla) [inline src/components/WorkOrder/MobileWorkingWorkOrderView.js:175]: what is the reason for adding this?

**Review by humulla — APPROVED** [review]

File: src/components/WorkOrder/MobileWorkingWorkOrderView.js (hunk 3 above)
Comment (LBHCallumM) [reply src/components/WorkOrder/MobileWorkingWorkOrderView.js:175]: Added a description param, but the other forms dont have the field yet.

File: cypress/integration/work-order/attend/close-by-proxy.spec.js
Comment (Duslerke) [inline cypress/integration/work-order/attend/close-by-proxy.spec.js:1]: I assume this is a TODO.

**Review by Duslerke — APPROVED** [review]
Good PR!

File: src/components/WorkOrders/SummaryCloseWorkOrder.js [hunk 4]
```diff
@@ -149,21 +149,32 @@ const SummaryCloseWorkOrder = ({
...
                 />
               </TBody>
             </Table>
           </>
         )}
 
-        <PrimarySubmitButton label="Confirm and close" />
+        <div style={{ margin: '30px 0 0 0', display: 'flex' }}>
+          {' '}
+          <PrimarySubmitButton
```
Comment (adamtry) [inline src/components/WorkOrders/SummaryCloseWorkOrder.js:166]: Not really a primary submit button if it's a secondary "go back" button. Could use another component if available or rename if it's not too disruptive?

**Pull request merged**